torch/testing/_internal/common_methods_invocations.py

```diff
@@ -895,6 +895,26 @@ def __call__(self, *args, **kwargs):
         """Calls the function variant of the operator."""
         return self.op(*args, **kwargs)
 
+    def __str__(self):
+        class_name = self.__class__.__name__
+        indent = (len(class_name) + 1) * " "
```

Comment:
Some of the OpInfo class names are quite long; maybe we should just newline and indent a fixed amount (2 spaces, perhaps?)